samples/bluetooth/mesh/common/smp_bt.c

```diff
@@ -142,6 +144,15 @@ int smp_service_adv_init(void)
 
 int smp_dfu_init(void)
 {
+	if (IS_ENABLED(CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN)) {
+		int err;
+
+		err = smp_bt_auth_init();
+		if (err) {
+			printk("Can't authentication for SMP (err: %d)\n", err);
```

Comment:
Shouldn't initialization go out here? Seems authentication initialization is failed, but everything goes further.